### PR TITLE
DEV-12704 update user image contoller to allow admins to change profi…

### PR DESCRIPTION
**Pull request by AndrewLngdn**
…le pictures

### CODE REVIEW

**Comment by alpinebuilder** [comment]
Can one of the admins verify this patch?

**Comment by AndrewLngdn** [comment]
Hey Kevin, this one should be quick and merge-able

File: app/controllers/user_images_controller.rb [hunk 1]
```diff
@@ -8,8 +7,7 @@ def load_entity
   end
 
   def authorize_create!
-    # Remove this method when 'authorize_create!' is taken out of
-    # ImagesController
+    Authority.authorize! :update, @user, current_user, { :or => :current_user_is_referenced_user }
```
Comment (kevin-alpine) [inline app/controllers/user_images_controller.rb]: Ok I loaded this PR up in my dev instance and tested it out.

It works:
* admins can change anyone's profile image
* nonadmins can only change their own

... one comment, I tried removing this bit, and, the behavior stayed exactly the same, is it necessary?

`{ :or => :current_user_is_referenced_user }`

Comment (kevin-alpine) [inline app/controllers/user_images_controller.rb]: I even hacked Backbone to allow my non-admin user to attempt to change someone else's image, and my attempt at hax0ring was rejected, correctly!

Comment (AndrewLngdn) [inline app/controllers/user_images_controller.rb]: I'll look into this in 5.7, good find!

**Pull request merged**